microsite/data/plugins/periskop.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Periskop
+author: Periskop
+authorUrl: https://periskop.io/
+category: Monitoring
+description: Periskop is a pull-based, language agnostic exception aggregator for microservice environments.
+documentation: https://github.com/backstage/backstage/tree/master/plugins/periskop
+iconUrl: https://raw.githubusercontent.com/periskop-dev/periskop/master/docs/assets/periskop-logo.png
```

Comment:
this is a 404 for me 🤔

Reply:
This is now merged :) https://github.com/periskop-dev/periskop/pull/192
The URL should be available 🤞